src/sbi_cove.adoc

```diff
@@ -761,30 +769,66 @@ enum tsm_state {
     TSM_READY = 2
 };
 
+#define COVE_TSM_CAP_PROMOTE_TO_TVM       0x1
+#define COVE_TSM_CAP_LOCAL_ATTESTATION    0x2
+#define COVE_TSM_CAP_AIA                  0x3
+#define COVE_TSM_CAP_DYN_PAGE_CONVERSION  0x4
+
 struct tsm_info {
     /*
-     * The current state of the TSM (see tsm_state enum above).
-     * If the state is not TSM_READY, the remaining fields are invalid and will
-     * be initialized to 0.
+     * The current state of the TSM (see `tsm_state` enum above).
+     * If the state is not `TSM_READY`, the remaining fields are invalid and
+     * will be initialized to `0`.
      */
     uint32_t tsm_state;
-    /* Version number of the running TSM. */
+    /* 
+     * Identifier of the TSM implementation. 
+     * It is intended to distinguish among different TSM implementations,
+     * potentially managed by different organizations, that might target
+     * different deployment models and, thus, implement subset of CoVE spec.
+     */
+    uint32_t tsm_impl_id;
+    /* 
+     * Version number of the running TSM. 
+     */
     uint32_t tsm_version;
+    /* 
+     * CoVE features supported by the running TSM.
+     * Enabled bit `i` indicates that the TSM and underly supports capability `i` and there
+     * is hardware support.
```

Comment:
The comment sounds bit ambiguous to me w.r.t "there is hardware support". I think something along the lines is sufficient.
"A bitmask of CoVE features supported by the running TSM." - We can refer to the capability table for additional clarification.

Reply:
The reason why I wrote it like this is that there are cases when software (TSM) and hardware support different set of features. The TSM must report to OS/VMM only features that it can use, i.e., the ones that are supported by software and hardware.

For example, take a platform with hardware support for AIA. If someone runs there a TSM that does not implement support for AIA, OS/VMM would have to use only TSM's legacy interrupt handling.

Reply:
I added a reference to the table as suggested.